src/app/component/user/components/habit/all-habits/all-habits.component.ts

```diff
@@ -143,4 +148,18 @@ export class AllHabitsComponent implements OnInit, OnDestroy {
     this.currentPage = 0;
     this.elementsLeft = true;
   }
+
+  public checkIfAssigned() {
+    this.habitAssignService.getAssignedHabits()
+    .pipe(take(1))
+    .subscribe((response: Array<HabitAssignInterface>) => {
+      response.map((assigned) => {
```

Comment:
Consider using "forEach" instead of "map" as its return value is not being used here.

Reply:
Done